tests/app/celery/test_scheduled_tasks.py

```diff
@@ -624,3 +631,27 @@ def test_trigger_link_does_nothing_if_cbc_proxy_disabled(
         trigger_link_tests()
 
     assert mock_trigger_link_test.called is False
+
+
+@freeze_time('2021-07-19 15:50')
+@pytest.mark.parametrize('status, finishes_at, final_status', [
+    (BroadcastStatusType.BROADCASTING, '2021-07-19 16:00', BroadcastStatusType.BROADCASTING),
+    (BroadcastStatusType.BROADCASTING, '2021-07-19 15:40', BroadcastStatusType.COMPLETED),
+    (BroadcastStatusType.BROADCASTING, None, BroadcastStatusType.BROADCASTING),
```

Comment:
Not relevant to this PR, just out of curiosity, but is this a valid state where we're broadcasting but we haven't set a `finishes_at`?

Reply:
Good question. While [we do set this time in the Admin app](https://github.com/alphagov/notifications-admin/blob/da976696747aa982d56e54eac3e14de7c64f6583/app/models/broadcast_message.py#L255), there's [nothing actually stopping the field from being `None`](https://github.com/alphagov/notifications-api/blob/master/app/broadcast_message/rest.py#L144). Although the Admin app is currently the only way to approve a broadcast, I wanted to make sure this code didn't misbehave if in future we have some other way of approving that doesn't set an expiry.